### PR TITLE
Use touch events when pointer events are not supported

### PR DESCRIPTION
**Pull request by amedina**
Since Safari does not support pointer events, modify DragObserver to work with touch events in such a case.

### CODE REVIEW

File: amp-pwa-reader/src/js/DragObserver.js [hunk 1]
```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
 
   _start (event) {
 
-    // store event for re-use
```
Comment (pbakaus) [inline amp-pwa-reader/src/js/DragObserver.js]: Could you please bring this comment back?

File: amp-pwa-reader/src/js/DragObserver.js [hunk 2]
```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
...
-    this.eventDown = event;
+    if (event.pageX) {
+      this.eventDown = event;
+    } else {
+      this.eventDown = event.targetTouches[0];
```
Comment (pbakaus) [inline amp-pwa-reader/src/js/DragObserver.js]: as noted, this is probably not needed as pointer-events seem to include pageX/pageY on the top level object nowadays.

File: amp-pwa-reader/src/js/DragObserver.js [hunk 3]
```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
...
+    }
 
     this.__move = (e) => this._move(e);
     this.__stop = (e) => this._stop(e);
-    document.addEventListener('pointermove', this.__move, { passive: true });
-    document.addEventListener('pointerup', this.__stop, { passive: true });
+
+    document.addEventListener(
+        this._supportsPointerEvents ? 'pointermove' : 'touchmove',
+        this.__move, { passive: true }
```
Comment (pbakaus) [inline amp-pwa-reader/src/js/DragObserver.js]: Use either new lines for all arguments, or none, but don't mix :)

File: amp-pwa-reader/src/js/DragObserver.js [hunk 4]
```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
...
+    );
+    document.addEventListener(
+        this._supportsPointerEvents ? 'pointerup' : 'touchend',
+        this.__stop, { passive: true }
```
Comment (pbakaus) [inline amp-pwa-reader/src/js/DragObserver.js]: ditto (re: new line style)

File: amp-pwa-reader/src/js/DragObserver.js [hunk 5]
```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
...
+    );
 
   }
 
   _move (event) {
 
+    if (event.pageX) {
+      this.eventMove = event;
+    } else {
+      this.eventMove = event.targetTouches[0];
```
Comment (pbakaus) [inline amp-pwa-reader/src/js/DragObserver.js]: same here

File: amp-pwa-reader/src/js/DragObserver.js (hunk 1 above)
Comment (amedina) [inline amp-pwa-reader/src/js/DragObserver.js]: Done.

File: amp-pwa-reader/src/js/DragObserver.js (hunk 2 above)
Comment (amedina) [inline amp-pwa-reader/src/js/DragObserver.js]: Done.

File: amp-pwa-reader/src/js/DragObserver.js (hunk 3 above)
Comment (amedina) [inline amp-pwa-reader/src/js/DragObserver.js]: Done.

File: amp-pwa-reader/src/js/DragObserver.js (hunk 4 above)
Comment (amedina) [inline amp-pwa-reader/src/js/DragObserver.js]: Done.

File: amp-pwa-reader/src/js/DragObserver.js (hunk 5 above)
Comment (amedina) [inline amp-pwa-reader/src/js/DragObserver.js]: Done.

**Pull request merged**